--- BitTorrent-4.1.7/makei18n.sh.orig	2005-11-04 12:51:16.000000000 +0100
+++ BitTorrent-4.1.7/makei18n.sh	2005-11-04 12:51:16.000000000 +0100
@@ -1,5 +1,5 @@
 APP_NAME="bittorrent"
-LANGUAGES=`python language_codes.py`
+LANGUAGES=`cd po; ls *.po | sed -e 's/\.po//'`
 MESSAGES_PO="messages.pot"
 
 rm -f $APP_NAME.lis
--- BitTorrent-4.1.7/BitTorrent/__init__.py.orig	2005-11-04 13:06:30.000000000 +0100
+++ BitTorrent-4.1.7/BitTorrent/__init__.py	2005-11-04 13:25:11.000000000 +0100
@@ -36,9 +36,9 @@
     'cs'   :u'Čeština'              ,    'de'   :u'Deutsch'              ,
     'en'   :u'English'              ,    'es'   :u'Español'              ,
     'es_MX':u'Español de Mexico '   ,    'fr'   :u'Français'             ,
-    'gr'   :u'Ελληνικά'             ,    'hu'   :u'Magyar'               ,
+    'el'   :u'Ελληνικά'             ,    'hu'   :u'Magyar'               ,
     'it'   :u'Italiano'             ,    'nl'   :u'Nederlands'           ,
-    'nb_NO':u'Norsk bokmål'         ,    'pl'   :u'Polski'               ,
+    'nb'   :u'Norsk bokmål'         ,    'pl'   :u'Polski'               ,
     'pt'   :u'Português'            ,    'pt_BR':u'Português do Brasil'  ,
     'ro'   :u'Română'               ,    'ru'   :u'Русский'              ,
     'sk'   :u'Slovenský'            ,    'sl'   :u'Slovensko'            ,
@@ -50,21 +50,18 @@
 
 unfinished_language_names = {
     'ar'   :u'العربية'       ,    'eo'   :u'Esperanto'            ,
-    'eu'   :u'Euskara'              ,    'et'   :u'Eesti'                ,
+    'et'   :u'Eesti'                ,
     'fi'   :u'Suomi'                ,    'ga'   :u'Gaeilge'              ,
-    'he_IL':u'עברית'                ,    'hr'   :u'Hrvatski'             ,
-    'hy'   :u'Հայերեն'       ,    'in'   :u'Bahasa indonesia'     ,
+    'he'   :u'עברית'                ,    'hr'   :u'Hrvatski'             ,
+    'hy'   :u'Հայերեն'       ,
     'ja'   :u'日本語'            ,    'ka'   :u'ქართული ენა',
     'ko'   :u'한국어'            ,    'lt'   :u'Lietuvių'        ,
     'ms'   :u'Bahasa melayu'        ,    'ml'   :u'Malayalam'            ,
     'sq'   :u'Shqipe'                ,    'th'   :u'ภาษาไทย'              ,
     'tlh'  :u'tlhIngan-Hol'         ,    'uk'   :u'Українська'           ,
-    'hi'   :u'हिन्दी'                  ,    'cy'   :u'Cymraeg'              ,
-    'is'   :u'Íslenska'             ,    'nn_NO':u'Norsk Nynorsk'        ,
-    'te'   :u'	తెలుగు'              ,
     }
 
-#language_names.update(unfinished_language_names)
+language_names.update(unfinished_language_names)
 
 languages = language_names.keys()
 languages.sort()
